src/indigo/components/SearchBar.js

```diff
@@ -29,6 +29,15 @@ class SearchBar extends React.Component {
     this.props.onSubmit();
   }
 
+  renderRecommendedLink(linkElement, index) {
+    return (
+      <span className="searchbar-recommended-link" key={index}>
+        <Icon.Search className="searchbar-recommended-link-icon icon-size-16" />
+        {linkElement}
```

Comment:
@ujovlado Domnivam se ze zatim je tohle optimalni reseni, ale je ok to takhle parametrizovat? Jde mi o to, jestli to neudelat prisnejsi = renderovat odkaz z poslanych parametru. Nebo naopak volnejsi - do radky mozno cpat i vic veci.